### PR TITLE
Ecs cluster dropdown

### PR DESCRIPTION
**Pull request by dkirillov**
@BrunoCarrier

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,4 @@ README.html
 es-tmp/
 clouddriver-oracle-bmcs/bmcs-sdk
 gatling.conf
+*.DS_Store
```
Comment (BrunoCarrier) [inline .gitignore]: don't mind having it here, but you probably will want to add this to your global gitignore at `~/.gitignore`

File: clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/RoleController.java [hunk 2]
```diff
@@ -38,16 +37,9 @@
...
-
-    //TODO(Bruno Carrier) - I am sure we can make a nice java Stream that does it more elegantly
-    for (RoleProvider roleProvider: roleProviders) {
-      if (roleProvider.getCloudProvider().equals(provider)) {
-        result.addAll(roleProvider.getAll(account, region));
-      }
-    }
-
-    return result;
+    return roleProviders.stream()
```
Comment (BrunoCarrier) [inline clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/RoleController.java]: Good job

File: .gitignore (hunk 1 above)
Comment (BrunoCarrier) [reply .gitignore]: Actually please remove it as we wont be able to merge that upstream, and update your global gitignore

Comment (dkirillov) [reply .gitignore]: done.

**Comment by BrunoCarrier** [comment]
LGTM +2

**Pull request merged**